experimental/GModule/Misc.jl

```diff
@@ -34,6 +68,7 @@ function Hecke.number_field(::QQField, a::qqbar; cached::Bool = false)
 end
 
 Base.getindex(::QQField, a::qqbar) = number_field(QQ, a)
+Base.getindex(::QQField, a::Vector{qqbar}) = number_field(QQ, a)
```

Comment:
maybe also
```suggestion
Base.getindex(::QQField, a::Vector{qqbar}) = number_field(QQ, a)
Base.getindex(::QQField, a::qbar, as::qqbar...) = number_field(QQ, [a, as...])
```
for interactive use: `QQ[s,c]`